### PR TITLE
Update and improve Turkish translations

### PR DESCRIPTION
**Pull request by bitigchi**

### CODE REVIEW

File: po/tr.po [hunk 1]
```diff
@@ -1137,12 +1137,12 @@ msgstr "Kaydetme iptal edildi."
...
 #, c-format
 msgid "Saved article to %s"
-msgstr "Yazı %s' konumuna kaydedildi"
+msgstr "Yazı, %s' konumuna kaydedildi"
 
 #: src/itemlistformaction.cpp:1389 src/itemviewformaction.cpp:569
 #, c-format
 msgid "Error: couldn't save article to %s"
-msgstr "Hata: Yazı %s' konumuna kaydedilemedi"
+msgstr "Hata: Yazı, %s' konumuna kaydedilemedi"
```
Comment (dennisschagt) [inline po/tr.po:1145]: This translation (and the one above) have only a single quote after the `%s` without an opening quote.
Is that intentional?

**Review by dennisschagt — APPROVED** [review]
I'm not familiar with the Turkish language, but the changes look good to me

**Comment by bitigchi** [comment]
> @dennisschagt approved this pull request.
> I'm not familiar with the Turkish language, but the changes look good to me
> 
> In po/tr.po:
> 
> >  
>  #: src/itemlistformaction.cpp:1389 src/itemviewformaction.cpp:569
>  #, c-format
>  msgid "Error: couldn't save article to %s"
> -msgstr "Hata: Yazı %s' konumuna kaydedilemedi"
> +msgstr "Hata: Yazı, %s' konumuna kaydedilemedi"
> This translation (and the one above) have only a single quote after the %s without an opening quote.
> Is that intentional?
> 
That’s an error, that quote is not supposed to be there. I’ll fix it tomorrow.

**Comment by dennisschagt** [comment]
Looks good, thank you!

**Pull request merged**